packages/flutter/lib/src/gestures/arena.dart

```diff
@@ -49,8 +49,11 @@ class GestureArenaEntry {
   ///
   /// It's fine to attempt to resolve a gesture recognizer for an arena that is
   /// already resolved.
-  void resolve(GestureDisposition disposition) {
-    _arena._resolve(_pointer, _member, disposition);
+  ///
+  /// If a [bid] is provided, the highest bid over 1.0 will be awarded victory upon
```

Comment:
1.0 = accept threshold. It’s the same reason as your question above. If you set value of at least 1.0, it means you would like to accept it. So any gesture recognizer which before would call accept at some threshold of movement, instead make sure the value is 1.0 at that point for the same behaviour. So if panSlop is 24.0, that’s equivalent to having bid of 1.0 when pan magnitude is 24.0 and bid of 2.0 when pan magnitude is 48.0.

It might seem kind of useless to send the accept when bid is below 1.0. But personally I have a fork where I reduced the threshold to accept from 1.0 to 0.5, it provides really responsive behaviour, but I don’t know if flutter team would accept it, as it’s sort of breaking with previous behaviour. 

It’s true the bidding curve could cause some issues with a new gesture recognizer. If you always bid 2.0, you are going to win each time. The strategy should to create your curve should be as follows:

1. Identify acceptance criteria (movement in a certain direction with a minimum magnitude)
2. Make the acceptance point have bid = 1.0
3. Linear fit from bid = 0.0 at no movement